.env.dev

```diff
@@ -1,3 +1,4 @@
 FRONTEND_URL="localhost:5000"
 GA_API_EMAIL_ADDRESS="google.service.account@example.com"
 GA_PRIVATE_KEY_PATH="path/to/private.key"
+PUBLIFY_HOSTNAME="localhost:4000"
```

Comment:
Might be nice if publify defaulted to this locally (or, at least, something that's not :3000).

@baphled wyt?